src/test/suite/extensionFunctions.test.ts

```diff
@@ -0,0 +1,44 @@
+import * as assert from 'assert';
+import * as vscode from 'vscode';
+import {extensionFunctions} from '../../extensionFunctions';
+
+suite('Extension Functions Test Suite', () => {
+
+	test('send text to terminal', async function() {
+		const testTerminalName = 'testTerminal';
+		var terminal : vscode.Terminal | undefined = extensionFunctions.findTerminal(testTerminalName);
+		assert.equal(terminal, undefined);
+
+		await extensionFunctions.sendTerminalText(testTerminalName, "testText");
+		terminal = extensionFunctions.findTerminal(testTerminalName);
+		assert.notEqual(terminal, undefined);
+
+		// open to ideas on how to check to see that a message was actually committed to the terminal
```

Comment:
we can use "side-effects", like the text sent to the terminal is to create a file somehwere and then we check that the file is created. But it seems a bit too effort for the value. I prefer the idea to wait for listener outputchannel content pointed by Lars to be in VS Code APi.